Office2019Suite-SKUless/OfficeSuiteSKULessVersionProvider.py

```diff
@@ -45,11 +46,10 @@ class OfficeSuiteSKULessVersionProvider(Processor):
     def get_version(self, installertype, FEED_URL):
         """Parse the macadmins.software/versions.xml feed for the latest O365 version number"""
         try:
-            raw_xml = urllib2.urlopen(FEED_URL)
-            xml = raw_xml.read()
-        except BaseException as e:
+            xml = self.download(FEED_URL)
+        except Exception as e:
             raise ProcessorError("Can't download %s: %s" % (FEED_URL, e))
-        version = ''
```

Comment:
this is so we return emptystring instead of None if a version can't be found (I know it looks unused, but it's a type difference that we don't want to cause an error in autopkg core with), please reinstate

Reply:
Done